packages/tooling/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.ts

```diff
@@ -59,7 +69,11 @@ export class HTMLRenderLayerNavigation extends HTMLRenderLayer {
             this.handleWindowChange
         );
 
-        this.resizeDetector.observe(document.body);
+        this.resizeDetector.observe(
+            (this.resizeobserverselector
+                ? document.querySelector(this.resizeobserverselector)
```

Comment:
Just to be safe, rather than referencing `document` here, you may want to call `this.getRootNode()`. That will get you the document if the component is in the light dom but if it happens to be used inside of another shadow dom, it will get you the shadowRoot. You should be able to perform the querySelector on either (though you might have to cast in TypeScript).

Reply:
Done.